### DISCUSSION TITLE
What is vvencFFapp?

### DISCUSSION THREAD

**Discussion by Discograp** [staxrip/staxrip#1100] [Q&A] [answered]
what is it?
is this the new 266 codec?

![266](https://github.com/staxrip/staxrip/assets/95682410/89df97fb-95af-4eb4-a071-221199e29050)

**Comment by Dendraspis** [comment, marked as answer]
1. ![image](https://github.com/staxrip/staxrip/assets/50659978/37ded3b0-3860-429b-bf7e-8ee4fe196b0d)
2. https://github.com/fraunhoferhhi/vvenc

**Comment by Discograp** [comment]
ahh - improved 265 - processes more pixels than 265, is av1 not more efficient?

![266](https://github.com/staxrip/staxrip/assets/95682410/ce3c8498-b3c4-4a07-b702-4a88b23198eb)
